library/std/src/sys/pal/sgx/thread.rs

```diff
@@ -45,18 +45,12 @@ mod task_queue {
         }
     }
 
-    #[cfg_attr(test, linkage = "available_externally")]
-    #[export_name = "_ZN16__rust_internals3std3sys3sgx6thread15TASK_QUEUE_INITE"]
-    static TASK_QUEUE_INIT: Once = Once::new();
     #[cfg_attr(test, linkage = "available_externally")]
     #[export_name = "_ZN16__rust_internals3std3sys3sgx6thread10TASK_QUEUEE"]
-    static mut TASK_QUEUE: Option<Mutex<Vec<Task>>> = None;
+    static TASK_QUEUE: Mutex<Vec<Task>> = Mutex::new(Vec::new());
 
-    pub(super) fn lock() -> MutexGuard<'static, Vec<Task>> {
-        unsafe {
-            TASK_QUEUE_INIT.call_once(|| TASK_QUEUE = Some(Default::default()));
-            TASK_QUEUE.as_ref().unwrap().lock().unwrap()
-        }
+    pub(super) fn tasks() -> MutexGuard<'static, Vec<Task>> {
```

Comment:
Why change the name of the function? It's not clear anymore in usage sites that this is returning a mutex guard, which complicates lock analysis.

Reply:
The reasoning was that `lock` doesn't tell the developer much more than it's locking something. Looking at it again, that's not really an issue here as we always explicitly mention the module name as well. I've removed the commit that renamed the function.